doc/nrf/known_issues.rst

```diff
@@ -215,7 +215,7 @@ Thread
 .. rst-class:: v1-5-0 v1-4-2 v1-4-1 v1-4-0
 
 KRKNWK-6848: Reduced throughput
-  Performance testing for :ref:`NCP sample <ot_ncp_sample>` shows a decrease of throughput of around 10-20% compared with the standard OpenThread.
+  Performance testing for :ref:`NCP sample <ot_coprocessor_sample>` shows a decrease of throughput of around 10-20% compared with the standard OpenThread.
```

Comment:
In the [documentation ](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thread_architectures.html)we distinguish `System-on-Chip designs` and `Co-processor designs`.
```suggestion
  Performance testing for :ref:`NCP sample <ot_coprocessor_sample>` shows a decrease of throughput of around 10-20% compared with the System-on-Chip design.
```

Reply:
If it's too late to change that please ignore this nit.

Reply:
I believe by "standard OpenThread" the author meant OpenThread with raw nrf528xx platform port.